### PR TITLE
Feedback widget

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Add `focus-trap-react` package
- Migrate `useKeyPress` hook
- Migrate `useOnClickOutside` hook
- Update `FeedbackWidget` component
- Implement `FeedbackWidget` inside `RootLayout`

Cherry-picks bc400d674 from #152, implementing `useSurvey` hook

## Related Issue
https://www.notion.so/efdn/Add-Feedback-widget-c99e6d8e04b0425ca6769c7ddf63ba7d?pvs=4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@emotion/styled": "^11.11.0",
     "@storybook/react": "^7.3.2",
     "clipboard": "^2.0.11",
+    "focus-trap-react": "^10.2.3",
```
Comment (minimalsm) [inline package.json:24]: Where is this being used?

Comment (wackerow) [reply package.json:24]: @minimalsm 
https://github.com/ethereum/ethereum-org-fork/blob/6c16af52680bc7636c8b14a1c1fef1baf7c3fdb7/src/components/FeedbackWidget.tsx#L2
https://github.com/ethereum/ethereum-org-fork/blob/6c16af52680bc7636c8b14a1c1fef1baf7c3fdb7/src/components/FeedbackWidget.tsx#L186

or in canonical:
https://github.com/ethereum/ethereum-org-website/blob/9163e6b7a3063c4ccec282c63b39685b60abe667/src/components/FeedbackWidget.tsx#L189

**Review by pettinarip — CHANGES_REQUESTED** [review]
I see this translation without parsing in the FeedbackCard. This happens when only the `t` function is used. For rich text content, we have to parse the content with `htmr` which is implemented in the `Translation` component.

image

**Review by pettinarip — APPROVED** [review]
Nice job @wackerow, looks great.

I've pushed a commit to fix the not parsed translation and some conflicts with dev.

**Review by nhsz — APPROVED** [review]
LGTM

**Pull request merged**